packages/matchbox/src/components/Page/Page.js

```diff
@@ -47,35 +47,41 @@ function Subtitle({ subtitle }) {
   return <div>{subtitle}</div>;
 }
 
-// TODO verify this with Popover and Actionlist tickets
 function SecondaryActions({ actions = [], hasPrimaryAction }) {
   const visibleActions = React.useMemo(() => filterByVisible(actions), [actions]);
+  const [isOpen, setIsOpen] = React.useState(false);
 
   if (!visibleActions.length) {
     return null;
   }
 
   return (
-    <Popover
-      bottom
-      left
-      trigger={
-        <Box position="relative" mr={hasPrimaryAction ? '200' : ' 0'}>
+    <Box position="relative" mr={hasPrimaryAction ? '200' : ' 0'}>
+      <Popover
+        bottom
+        id="page-secondary-actions"
+        left
+        onClose={() => setIsOpen(false)}
+        open={isOpen}
+        trigger={
           <Button
-            outline
+            aria-describedby="page-secondary-actions"
```

Comment:
@jonambas Just saw this, but looks like it's already been merged - technically `aria-describedby` isn't really appropriate in this scenario. `aria-controls` would probably make more sense - the `<Expandable/>` component would be a good reference for what that should look like.

Reply:
Eek, thanks that makes sense. I'll cut a follow up ticket